### ISSUE TITLE
Update README.md to include "artName" property for include.js

### ISSUE THREAD

**Issue by donmarvex** [zero-to-mastery/CSS-Art#273]
From recent pull requests, contributors have omitted this property because it isn't explicitly stated in the README.md file.

The current documentation has this below. 
```JavaScript
let cards = [
  //  Add your card in this section
  {
    pageLink: "./Art/Joy/triangle/triangle.html",
    imageLink: "./Art/Joy/triangle/triangle.gif",
    author: "Joy",
    githubLink: "https://github.com/royranger",
  },
]
```

However, in the _**include.js**_ file, five properties are used and artName is included among these properties;

```JavaScript
`<img class="art-image" src='${c.imageLink}' alt='${c.artName}' />` +
```

In order to prevent future contributors from omitting the property, it should be updated in the documentation.
```JavaScript
let cards = [
  //  Add your card in this section
  {
    artName: "My art name here",
    pageLink: "./Art/Joy/triangle/triangle.html",
    imageLink: "./Art/Joy/triangle/triangle.gif",
    author: "Joy",
    githubLink: "https://github.com/royranger",
  },
]
```
I can raise a PR to update the docs or we could have another contributor update the docs as a good first issue practice. Please review @sophiabrandt

**Comment by sophiabrandt** [comment]
@donmarvex Hello, I'd be happy to accept your pull request if you have time. Thank you! 👍

**Comment by donmarvex** [comment]
> @donmarvex Hello, I'd be happy to accept your pull request if you have time. Thank you! 👍

Hello @sophiabrandt I have made a pull request with the changes. Thank You

**Comment by sophiabrandt** [comment]
closed via #277 by @donmarvex . Thank you.
